lib/tasks/metrics.rake

```diff
@@ -0,0 +1,6 @@
+namespace :metrics do
+  desc "Collects metrics from Forem instances"
+  task overview: :environment do
+    puts Ahoy::Event.overview_metrics_to_csv
```

Comment:
I'm not sure if this happens in production as well but when a rake task is loaded, a bunch of warnings are printed on the standard output, and with `rake metrics:overview > filename.csv` those warnings will end up in the CSV file. On my local installation this is what I found in the file:

```csv
Unset ENV variable: IMGPROXY_KEY.
Unset ENV variable: IMGPROXY_SALT.
DEPRECATION WARNING: Initialization autoloaded the constants RateLimitChecker, ForemStatsDriver, ForemStatsDrivers, ActsAsForemStatsDriver, and ForemStatsDrivers::DatadogDriver.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload RateLimitChecker, for example,
the expected changes won't be reflected in that stale Class object.

These autoloaded constants have been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.
 (called from <main> at /forem/config/environment.rb:5)
Unset ENV variable: RELEASE_FOOTPRINT.
** [Honeybadger] Initializing Honeybadger Error Tracker for Ruby. Ship it! version=4.7.3 framework=rails level=1 pid=49260
  [1m[36mProfileField Load (0.8ms)[0m  [1m[34mSELECT "profile_fields".* FROM "profile_fields" ORDER BY "profile_fields"."id" ASC LIMIT $1[0m  [["LIMIT", 1000]]
  ↳ app/models/profile.rb:55:in `refresh_attributes!'
DEPRECATION WARNING: Devise::Models::Authenticatable::BLACKLIST_FOR_SERIALIZATION is deprecated! Use Devise::Models::Authenticatable::UNSAFE_ATTRIBUTES_FOR_SERIALIZATION instead. (called from const_get at /forem/vendor/cache/devise-0cd72a56f984/lib/devise/models.rb:90)
  [1m[36mAhoy::Event Load (0.8ms)[0m  [1m[34mSELECT "ahoy_events".* FROM "ahoy_events" WHERE "ahoy_events"."name" = $1 ORDER BY "ahoy_events"."id" ASC LIMIT $2[0m  [["name", "Overview Link Clicked"], ["LIMIT", 1000]]
  ↳ app/models/ahoy/event.rb:17:in `block in overview_metrics_to_csv'
id,name,user_id,visit_id,time,target
1,Overview Link Clicked,,1,2021-03-17 15:56:51 UTC,https://example.com
```

Reply:
Good point, so it probably makes sense to save the data to a file rather than just printing it to the console!

Reply:
I think @vaidehijoshi made a solid suggestion - let's not overcomplicate things for now! This should probably just spit something human-readable out for us to copy/paste elsewhere!